SPECS/python-six/python-six.spec

```diff
@@ -53,11 +53,8 @@ python2 setup.py install --prefix=%{_prefix} --root=%{buildroot}
 python3 setup.py install --prefix=%{_prefix} --root=%{buildroot}
 
 %check
-easy_install_2=$(ls /usr/bin |grep easy_install |grep 2)
-$easy_install_2 pytest
-python2 test_six.py
```

Comment:
Double-checking: do we want to remove Python 2 tests in 1.0, since we still have that package there?

Reply:
yes.